civihr_default_theme/scss/components/_panel.scss

```diff
@@ -250,6 +250,7 @@ form {
   }
 }
 
+.my-details-view-panel,
```

Comment:
is this class auto generated by drupal?

Reply:
No it is a class I used to easily extend its styles to another files. Replaced with a placeholder selector.

Reply:
Ok so this is a custom class, then we need to follow BEM naming convention. Name it `.chr_panel--my-details__view__panel`